README.md

```diff
@@ -22,6 +22,7 @@
 - Tiny size
 - Beautiful
 - No classes
+- Encourage accessible code
```

Comment:
Sounds good, but what does this refer to?
I don't think we have rules that highlight `<img>` tags without an `alt` attr or similar stuff?

Reply:
Ah, probably the fact that we're styling HTML elements so it won't work for sites that are made only from `<div>` and `<span>`? 😄

Reply:
"Encourage semantic code" might be better? Anything built with water.css should be semantic, but not necessarily accessible if you've done some funky things.

Reply:
Yeah good point as well. And I'm still iffy on "encourage" instead of "encourages"